pandas/core/frame.py

```diff
@@ -4281,7 +4281,7 @@ def reset_index(self, level=None, drop=False, inplace=False, col_level=0,
             the index to the default integer index.
         inplace : bool, default False
             Modify the DataFrame in place (do not create a new object).
-        col_level : int or str, default 0
+        col_level : int, str, tuple, or list, default 0
```

Comment:
The tuple / list cases will need to be better documented here. I'd recommend a list

* int / str : ...
* tuple list : ...

Further, I don't think a tuple should be a valid here, as that's typically used to refer to a single key, not a sequence of keys.